_posts/en/newsletters/2020-11-25-newsletter.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Bitcoin Optech Newsletter #125'
+permalink: /en/newsletters/2020/11/25/
+name: 2020-11-25-newsletter
+slug: 2020-11-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a website that tracks miner support for
+taproot activation and includes our regular sections with popular
+questions and answers from the Bitcoin StackExchange, announcements of
+releases and release candidates, and notable changes to popular Bitcoin
+infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Website tracking miner support for taproot activation:** Bitcoin
+  Mining pool [Poolin][] has created the website
+  [TaprootActivation.com][] to help track miner support for activating
+  [taproot][topic taproot] (including [schnorr signatures][topic schnorr
+  signatures] and [tapscript][topic tapscript]).  As of this writing,
+  all pools listed on the site---who represent more than half of the
+  current network hash rate---have indicated that they will support
+  activation.  If this is reflective of the opinions of the remaining
+  pools, then it should be easy to accomplish the activation of taproot
```

Comment:
It seems to me that a pool that is in support would be a lot more responsive to the creation of such a site. Perhaps the possibility of a pool not being in favor, but not having chimed in yet, would be also worth a mention.

Reply:
I don't see an easy way to add this without making the sentence unweildly, and I'm hoping that  "If this is reflective" covers that possibility sufficiently enough that the sentence won't be considered inaccurate if some pools do end up opposing the change.

Reply:
Yeah, on revisiting, I still don't have a more concrete improvement suggestion. I'm a bit worried about how the website's relevance will be perceived by the less initiated reader, but it's pretty clear to the careful reader that the implementation and activation are still to be released. :+1: